chirp/drivers/ft450d.py

```diff
@@ -451,13 +494,14 @@ def _clone_in(self):
             else:
                 repeat = 1
             for _i in range(0, repeat):
-                data += self._read(block, blocks)
-                self.pipe.write(chr(CMD_ACK))
+                chunk = self._read(block, blocks)    # returns bytes()
+                data += chunk
+                self.pipe.write(bytes(chr(CMD_ACK), encoding='utf8'))

```

Comment:
This is wrong, you should not encode binary data as a UTF-8 string. What you want is either:

```
CMD_ACK = b'\x06'
self.pipe.write(CMD_ACK)
```
or
```
CMD_ACK = 0x06
self.pipe.write(bytes([CMD_ACK]))
```